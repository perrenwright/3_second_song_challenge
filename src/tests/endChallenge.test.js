import React from "react";
import { create } from "react-test-renderer";
import EndPage from "../components/endChallenge.js"


describe("endChallenge component", () => {
  test("Matches the snapshot", () => {
    const endChallenge = create(<EndPage />);
    expect(endChallenge.toJSON()).toMatchSnapshot();
  });
});
