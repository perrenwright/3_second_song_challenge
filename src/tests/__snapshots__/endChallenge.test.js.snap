// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`endChallenge component Matches the snapshot 1`] = `
<div
  className="LandingPage"
>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-text"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Return to challenges
    </span>
  </button>
  <font
    color="black"
  >
    <h1>
       
       your score is 
       
    </h1>
  </font>
</div>
`;
